README-Gitpod.md

```diff
@@ -0,0 +1,39 @@
+Token Contract in AssemblyScript - Gitpod version
+=================================================
+
+This is a stripped-down example where we'll demonstrate creating a token. We'll visit a page, sign in and use your browser's console to run a commands to initialize, send, and get the balance of a custom token.
+
+This README is specific to Gitpod and this example. For local development, please see [README.md](README.md).
+
+## Description
+
+This project contains implementation of token contract similar to [ERC20](https://theethereum.wiki/w/index.php/ERC20_Token_Standard).
+
+## Getting started
+
+Gitpod has taken care of installing all the necessary tools and dependencies. At the bottom of Gitpod is a terminal which will display a link to follow:
+```bash
+Server running at http://localhost:1234
+```
```

Comment:
Do you think it would make sense to open this link automatically for people using the `ports` section of `gitpod.yml`? https://www.gitpod.io/docs/config-ports/

Then we could remove this part from the docs, as well as the part below with the screenshot of the dialog that may appear

Reply:
we have ports on there already, copied the verbiage from `guest-book`